shared/uavobjectdefinition/hwdtfc.xml

```diff
@@ -0,0 +1,98 @@
+<xml>
+	<object name="HwDTFc" singleinstance="true" settings="true" category="HardwareSettings">
+		<description>Selection of optional hardware configurations.</description>
+
+		<field name="RcvrPort" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="PPM">
+			<options>
+				<option>Disabled</option>
+				<option>PPM</option>
```

Comment:
The available port options start with disabled and are alphabetized from there down on all the other targets.  Makes finding the desired option easier.  Not a big deal, but makes all targets consistent.